rascaline-c-api/src/lib.rs

```diff
@@ -42,22 +48,12 @@ impl log::Log for RascalLogger {
                 record.args());
             let cstr = CString::new(message).unwrap();
             unsafe {
-              GLOBAL_CALLBACK.unwrap()(record.level() as i32, cstr.as_ptr());
+                //let mut guard = GLOBAL_CALLBACK.lock().unwrap();
+                //let mut f = guard.unwrap();
+                GLOBAL_CALLBACK.lock().unwrap().unwrap()(record.level() as i32, cstr.as_ptr());
```

Comment:
first unwrap should be an `expect("mutex was poisonned")` or similar. We want to handle the second unwrap explicitly: if `GLOBAL_CALLBACK` is None, that means that the function pointer is NULL, and we don't want to call it. Instead we will want to skip sending the message since no logger was setup.